spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -42,17 +45,32 @@
           "affiliations": []
         }],
         "title": "#{generic_file.title.first}",
-        "additional_titles": [{
-          "title": "Secondary Title",
-          "type": "alternative_title",
-          "lang": "eng"
-        },
-        {
-          "title": "Tertiary Title",
-          "type": "alternative_title",
-          "lang": "eng"
-        }],
-        "formats": "application/pdf"
+        "additional_titles": [
+          {
+            "title": "Secondary Title",
+            "type": "alternative_title",
+            "lang": "eng"
+          },
+          {
+            "title": "Tertiary Title",
+            "type": "alternative_title",
+            "lang": "eng"
+          }
+        ],
+        "subjects": [
```

Comment:
As you will notice in here, the `identifier` field is left out entirely. I could not find a mapping quickly so I decided to get down what I could and then check back to see if there was somewhere to look these values up. [The documentation](https://inveniordm.docs.cern.ch/reference/metadata/#subjects-0-n) for this field listed it as optional (cardinality 0-1). @fenekku do you think this is something we should add to the export or can it be handled during import? I've got to believe there is some API for looking of LCSH and MESH terms, but I have no idea what to do for the keywords which are taken from GenericFile's tag field.

Reply:
I think this is something that should be done by the export if possible. DigitalHub has a local copy of MeSH terms (iirc) so it can use it to get the right identifier (it may also have local LCSH). For non-controlled terms a.k.a "keywords" those also go in subjects in Invenio but their scheme can be left empty.

All that being said, Lars is going over the vocabularies backend in InvenioRDM this month. Vocabularies are our generalization of the mechanism used for subjects terms. Do the best you can with the subjects for now, then mark it as blocked or make another ticket to revisit. We will have to revisit, once Lars' work is done to make sure we have the right structure. Let me know if that makes sense. We can take it to chat.